test/e2e/test_helm_install_upgrade.go

```diff
@@ -304,8 +300,10 @@ func TestHelmUpgrade(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			// Repository is different for OSS and Enterprise images and it should be set accordingly.
 			kgoImageRepository := "docker.io/kong/gateway-operator-oss"
+			kgoImageRepositoryNightly := "docker.io/kong/nightly-gateway-operator-oss"
 			if helpers.GetDefaultDataPlaneBaseImage() == consts.DefaultDataPlaneBaseEnterpriseImage {
 				kgoImageRepository = "docker.io/kong/gateway-operator"
+				kgoImageRepositoryNightly = "docker.io/kong/nightly-gateway-operator"
```

Comment:
Nit: could we extract these to `const`s?

Reply:
Fair enough. Given that we have the prod registries here as well let's leave the nightly ones as well here.